owncloudApp/src/main/java/com/owncloud/android/ui/preview/PreviewVideoActivity.kt

```diff
@@ -318,6 +333,8 @@ class PreviewVideoActivity : FileActivity(), Player.Listener, OnPrepareVideoPlay
     }
 
     override fun onPrepareMenu(menu: Menu) {
+
+
```

Comment:
Again these new blank lines why? 😁

Reply:
Removed too. I don't know why that lines are there. 😅